site/sass/components/_toc.scss

```
@@ -3,22 +3,23 @@
 // Breakpoints are tied to tech-detail layout, so before reusing it some adjustments may be necessary.
 
 .toc {
-  @include expandingNav($bkpt-tech-details--3-cols, 'toc', 1rem, 1rem);
+  $shadow-color: var(--grey-200);
+  $block-padding: 1.25rem;
+  @include expandingNav($bkpt-tech-details--3-cols, 'toc', 1.75rem, 1.33rem);
 
   @include mq($bkpt-tech-details--3-cols) {
     position: sticky;
     top: 6.5rem;
 
     &__summary {
-      padding-left: 1.75rem;
+      padding-left: $block-padding;
     }
 
     &__nav {
+      border-left: 1px solid $shadow-color;
```

```suggestion
      border-left: 2px solid $shadow-color;
```

Like we discussed, I think my bad here, this should be 2px. Otherwise looks great!